src/services/translation/TranslationService.js

```
@@ -32,18 +33,140 @@ class TranslationService {
+          // Removed credentials: 'same-origin' to ensure i18n files are fetched without authentication
+        });
+
+        if (!response.ok) {
+          throw new Error(`HTTP error! Status: ${response.status}`);
+        }
+
+        const translations = await response.json();
+        console.log(`Successfully loaded translations for ${lang}`);
+
+        // Cache the translations
+        this.translations[lang] = translations;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)